From bbcf7f0ee9530c5e5b9b6efa82d19ef66c0dfbc3 Mon Sep 17 00:00:00 2001
From: Krzysztof Niemiec <krzysztof.niemiec@intel.com>
Date: Wed, 29 Jul 2026 11:18:17 +0200
Subject: [PATCH 08190/15333] gpu/tests: Add a dedicated .kunitconfig

Add a dedicated .kunitconfig for running the GPU buddy allocator tests.

Signed-off-by: Krzysztof Niemiec <krzysztof.niemiec@intel.com>
Reviewed-by: Sebastian Brzezinka <sebastian.brzezinka@intel.com>
Reviewed-by: Andi Shyti <andi.shyti@linux.intel.com>
Reviewed-by: Krzysztof Karas <krzysztof.karas@intel.com>
Link: https://patch.msgid.link/20260729091816.27860-4-krzysztof.niemiec@intel.com
Signed-off-by: Maarten Lankhorst <dev@lankhorst.se>
---
 drivers/gpu/tests/.kunitconfig | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 drivers/gpu/tests/.kunitconfig

diff --git a/drivers/gpu/tests/.kunitconfig b/drivers/gpu/tests/.kunitconfig
new file mode 100644
index 000000000000..fd611b3dfe09
--- /dev/null
+++ b/drivers/gpu/tests/.kunitconfig
@@ -0,0 +1,2 @@
+CONFIG_KUNIT=y
+CONFIG_GPU_BUDDY_KUNIT_TEST=y
-- 
2.55.0

